repondeur/zam_repondeur/services/fetch/an/amendements.py

```diff
@@ -461,23 +480,56 @@ def build_url(
 
 class ANDerouleurData:
     """
-    Data extaction for Assemblée Nationale dérouleur
+    Data extraction for Assemblée Nationale dérouleur
     """
 
     def __init__(self, lecture: Lecture, content: dict):
         self.lecture = lecture
-        self.content = content
+        self.discussion_items = self._discussion_items(content)
+        self.position = self._position(self.discussion_items)
```

Comment:
Why not `positionS`? If I understand correctly it stores a dict with all the positions?

Reply:
I refactored all this part